homeworks/src/main/java/hw31/jdbc/SQLQuery.java

```diff
@@ -0,0 +1,28 @@
+package hw31.jdbc;
+
+public class SQLQuery {
```

Comment:
make this class final and add a private constructor, this would be the best practice as you should forbid the creation of this class because its purpose is to provide some static constants and nothing else. JFYI, the same should be done to manually created utility classes.